docs/index.asciidoc

```diff
@@ -86,6 +86,12 @@ Logstash `@timestamp` field.
 This configuration results in daily index names like
 +filebeat-{logstash_version}-{localdate}+.
 
+==== Event Metadata and the Elastic Common Schema (ECS)
+This plugin other than decoding `beat` events, adds some fields related to the deprecated `host`,
+containing the `hostname` provided by beats and `ip_address` which contains the remote address
+of the client's connection. These two fields are now moved in ECS compatible namespaces
+when the select compatibility mode is enabled, see <<plugins-{type}s-{plugin}-ecs_compatibility>>.
+
```

Comment:
What about something like this: 

> When <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is
enabled, this plugin adds {ecs-ref}[ECS] fields containing the `hostname` provided by beats and
the remote `ip_address` of the client's connection.

Reply:
I think that the meaning is shifted, from the sentence above seems like that the 2 fields are added only when ecs is enabled, but the plugin always add those fields, the only difference is that when ecs is enabled those fields are remapped

Reply:
I tried to rephrase it as:

> This plugin other than decoding `beat` events adds a couple of fields related to the event, the deprecated `host`
> which contains the `hostname` provided by beats and the `ip_address` containing the remote address
> of the client's connection. When <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is
> enabled these are now moved in ECS compatible namespaces.


LMKWDYT

Reply:
Thanks for the explanation. 
Would you consider changing the introduction slightly, to "In addition to decoding..."  or "When decoding..."

>When decoding `beats` events, this plugin adds two fields related to
the event: the deprecated `host` which contains the `hostname` provided by beats
and the `ip_address` containing the remote address of the client's connection.
When <<plugins-{type}s-{plugin}-ecs_compatibility,ECS compatibility mode>> is
enabled these are now moved in ECS compatible namespaces.

Reply:
Thanks for adding the other suggestions. Everything else looks as expected.